Terminal Commands.md

```diff
@@ -48,17 +67,56 @@ Some commands can be found in the `bin` directory.
 `>` - used to store the output of a task in some file(overwrites if file with same name is present) rather than displaying it on the terminal.<br/>
 `>>` - same task as `>` but does not overwrites just appends to the file with the same name.<br/>
 
-`cat` - used to open a file in terminal in read-only format
-
 `|` - piping is used to give the output of a command as input to another command ..for e.g `history | grep "find"` will search for "find" in the output of `history`
 
 `<command> | tee <filename>` - used to show output of `command` on terminal as well as writing the output to a file `filename`.<br/>
 
+`cat` - used to open a file in terminal in read-only format<br>
+`cat file1 > file2` - copies content of file1 one to file2<br>
+Unless you have infinite scrolling turned on (available in Profile Preferences -> Scrolling tab of the terminal), there is a limit to how many lines you can see on the screen.
+
+Piping the output using `less` allows the user to advance through the content by pressing SPACE, move backwards by pressing 'b' and quit using 'q'. Pressing ESC followed by SPACE allows you to scroll down one screen at a time.
```

Comment:
This is neither a trivia nor a command. Rather write it something like `(output of some command) | less` and then mention this stuff. Similarly for `more`.